docs/docs/intro.md

```diff
@@ -13,25 +13,31 @@ feature.
 
 ## Features
 
-At this point, ZMK is _missing_ more features than it has. Currently, the mostly working bits
+At this point, ZMK is _missing_ more features than it has. Currently, the working bits
+At this point, ZMK is still missing many features. Currently, the working bits
 include:
 
-- HID Over GATT (HOG) - This is the official term for BLE HID devices
-- Keymaps and layers with basic keycodes
-- Some initial work on one "behavior", Mod-Tap
-- Basic HID over USB
-- Basic consumer (media) keycodes.
-- Basic OLED display logic
-- Basic Split support
-- Encoders
+- Bluetooth connectivity (HID Over GATT (HOG) - This is the official term for BLE HID devices)
```

Comment:
Perhaps "Wireless connectivity via BLE HID Over GATT (HOG)"
We've tried to avoid using the word Bluetooth for this reason:
https://www.bluetooth.com/develop-with-bluetooth/marketing-branding/brand-enforcement-program/